Scoop/Controller/Home/HomeViewController.swift

```diff
@@ -169,24 +171,39 @@ class HomeViewController: UIViewController {
                     
                     if let date = dateFormatterGet.date(from: ride.departureDatetime) {
                         rideCopy.departureDatetime = dateFormatterPrint.string(from: date)
+                    }
+                    
+                    /// Separate a user's rides into those that are active and those that are pending
+                    var activeRidesIDs = []
+                    for ride in NetworkManager.shared.currentUser.rides {
+                        activeRidesIDs.append(ride.id)
+                    }
+
+                    let value = activeRidesIDs.contains { rideID in
+                        if rideID as! Int == rideCopy.id {
+                            return true
+                        } else {
+                            return false
+                        }
+                    }
+                    
+                    if value == true {
+                        self.activeRides.append(ride)
                     } else {
-                        print("There was an error decoding the datetime string")
+                        self.pendingRides.append(ride)
                     }
-                    self.activeRides.append(rideCopy)
                 }
+                
                 DispatchQueue.main.async {
                     self.tableView.reloadData()
                 }
+                
             case .failure(let error):
                 print(error)
```

Comment:
I would add some other message before the error and use string interpolation:
`print("Unable to blah blah blah: \(error.localizedDescription)"`
Personally, not a fan of using a localizedDescription because they don't tell you anything useful but I believe it is the standard